web/packages/teleport/src/Player/Player.tsx

```diff
@@ -121,3 +160,10 @@ const StyledPlayer = styled.div`
     overflow-y: hidden !important;
   }
 `;
+
+async function fetchSessionDuration(
```

Comment:
nit: our usual pattern is to define functions that interact with the API in `teleport/services`, I would move this to `teleport/services/recordings.tsx` instead of this component file